TimeCalc.java

```diff
@@ -1,5 +1,33 @@
 public class TimeCalc {
     public static void main(String[] args) {
-        // Replace this comment with your code
+        String time = args[0]; 
+        int minutesToAdd = Integer.parseInt(args[1]); 
+        int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1)); 
+        int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4)); 
+        int newMinutes = minutes + minutesToAdd ; 
+        while (newMinutes >= 60) {
+            newMinutes = newMinutes - 60; 
+            hours ++;
+            if (hours == 24) {
+                hours = 0; 
+            } 
+        }
+        String minutesOut = "" + newMinutes; 
+        if (newMinutes < 10) {
+            if (newMinutes == 0){

```

Comment:
redundant condition..
if (newMinutes == 0) 
the statement  `minutesOut = "0" + newMinutes;` is the same as `minutesOut = "00";`

The same with hours later..